docs/evaluate/temporal-cloud/pricing.mdx

```diff
@@ -275,8 +276,12 @@ Basic support has a monthly fee starting from $200.
 
 **Can I purchase Temporal Cloud through my Amazon, Azure, or Google Cloud Platform marketplace?**
 
-You can purchase Temporal Cloud credits in the AWS Marketplace.
-To learn more about our private AWS Marketplace offer, contact our team at [sales@temporal.io](mailto:sales@temporal.io).
+There's two ways to purchase Temporal Cloud through AWS Marketplace:
+
+- Pay-as-you-Go available [here](https://aws.amazon.com/marketplace/pp/prodview-hbcjz7wcl2mvu)
```

Comment:
Check all occurrances of Pay-as-yo-Go for casing and hyphens or spacing and ensure they are consistent. This one differs from the one that follows.

Reply:
Checking and making them all Pay-As-You-Go, which OpenAI says is correct. Thanks again.